src/cli/command_info.rs

```diff
@@ -42,16 +135,30 @@ pub fn execute_external_command(args: Vec<String>) -> miette::Result<()> {
 
         Ok(())
     } else {
-        // build the error message
-        // using the same style as clap's derived error messages
+        // Generate suggestions for similar commands
+        let suggestions = find_similar_commands(cmd, 0.6);
+
+        let mut error_msg = format!("No such command: `pixi {}`", cmd);
+
+        if !suggestions.is_empty() {
+            error_msg.push_str("\n\n");
+            if suggestions.len() == 1 {
+                error_msg.push_str(&format!("Did you mean '{}'?", suggestions[0]));
```

Comment:
I would like to suggest to use the same style how clap does it.

so you write a tip with only one command

```shell
error: unexpected argument 'ad' found

  tip: a similar subcommand exists: 'add'
```